packages/react-menu/package.json

```diff
@@ -43,10 +43,11 @@
     "react-test-renderer": "^16.3.0"
   },
   "dependencies": {
-    "@fluentui/theme": "^2.0.0-beta.13",
     "@fluentui/react-compose": "^1.0.0-beta.11",
+    "@fluentui/react-hooks": "^8.0.0-beta.10",
     "@fluentui/react-theme-provider": "^1.0.0-beta.19",
     "@fluentui/set-version": "^8.0.0-beta.1",
+    "@fluentui/theme": "^2.0.0-beta.13",
```

Comment:
I don't think that we should have this dependency

Reply:
May be `react-theme` should be there? 😼

Reply:
Reverted the package.json and fixed it all manually